openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1119,6 +1121,67 @@ Cache-Control: no-store
 }
 ```
 
+# Callback Endpoint {#callback_endpoint}
+
+This endpoint is used to receive notification from the Wallet whether credential has been successfully received in the Wallet or not. The Credential Issuer needs to return `callback_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL.
+
+This endpoint can be used after the Credential Issuer has sent Credential Response or Batch Credential Response. It enables the Credential Issuer to take subsequent actions after issuance, depending on whether the credential has been accepted and successully stored by the Wallet, rejected by the Wallet, or errors and other unforeseen circumstances have occurred during the Wallet's processing.
+
+Communication with the Callback Endpoint MUST utilize TLS.
+
+## Callback from the Wallet {#callback}
+
+The Wallet sends an HTTP POST request to the Callback Endpoint with the following parameters in the entity-body and using the `application/json` media type.
+
+* `issuer_state`: REQUIRED if the `issuer_state` parameter was present in the Credential Offer (see (#credential_offer)).
```

Comment:
that's redundant as it can be conveyed by the callback_id 

redundancies create new error cases, the issuer_state might not match all callback ids and/or the access token

Reply:
yes, it was pre-mature to add here before agreement on issue #79
```suggestion
```